getting-started/scaling-out.md

```diff
@@ -172,15 +162,22 @@ hypertable, you can use [`detach_data_node`][detach_data_node].
 SELECT detach_data_node('node1', hypertable => 'conditions');
 ```
 
-If a data node is storing data for a hypertable,
-then you will get an error.
+Note that you cannot detach a data node that still holds data for the
+hypertable. A common way to solve this issue is to use a retention
```

Comment:
"common"? We have not really had it in production. Suggest "You can solve this by using the retention period of the data and make sure..."

I think you should also have a link to a description about the retention period.

Reply:
Not sure what you mean by `description about the retention period.`? This is more of a concept enabled by drop_chunks.

Reply:
See update.